### ISSUE TITLE
[Enhancement]: DataNode need more higher test coverage

### ISSUE THREAD

**Issue by NaturalSelect** [cubefs/cubefs#2586]
### Contact Details

2145973003@qq.com

### Is there an existing issue for this?

- [X] I have searched all the existing issues

### What would you like to be added?

The test coverage of DataNode is too low(1.27％ from [codecov.io](https://app.codecov.io/gh/cubefs/cubefs) ).

### Why is this needed?

* Maybe we should make action to increase the test coverage of DataNode.
* It also can be a good first issue for those person who want to take part in CubeFS project.

**Comment by xiaochunhe** [comment]
yes, unit test is very important to the health of the system, we need to to gradually complete the unit tests
